test/uk/gov/hmrc/merchandiseinbaggage/pact/MibConnectorContractSpec.scala

```diff
@@ -153,14 +154,14 @@ class MibConnectorContractSpec extends BaseSpecWithApplication with CoreTestData
       interaction
         .description("check EoriNumber")
         .provided(s"checkEoriNumberTest")
-        .uponReceiving(GET, s"${checkEoriUrl}GB123")
+        .uponReceiving(GET, s"${config.mibCheckEoriUrl}GB123")
         .willRespondWith(200, Json.toJson(CheckResponse("GB123", true, None)).toString)
     )
     .addInteraction(
       interaction
         .description("findBy mib ref and eori")
         .provided(s"findByTestXXX${Json.toJson(findByDeclaration).toString}")
-        .uponReceiving(GET, s"$declarationsUrl?mibReference=${mibReference.value}&eori=${eori.value}")
+        .uponReceiving(GET, s"${config.mibDeclarationsUrl}?mibReference=${mibReference.value}&eori=${eori.value}")
```

Comment:
Can just be `.uponReceiving(GET, s"/declare-commercial-goods/declarations?mibReference=${mibReference.value}&eori=${eori.value}")`